examples/tpm/tpm_io.c

```diff
@@ -0,0 +1,198 @@
+/* tpm_io.c
```

Comment:
No, I do not want a copy of the example / reference tpm_io code in wolfSSH. That is duplicating a ton of code. This should reference the submodule example code...

Reply:
OK. Will fix.